### ISSUE TITLE
Update links in 2.0/2.1 HELP to new & latest techniques published by W3C in WCAG 2.2

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1696]
- [x] update URLs in all the existing HELP files (~170) to all the latest & new techniques and WCAG references to the newly published WCAG 2.2 resources such as [www.w3.org/WAI/WCAG22/Techniques/](https://www.w3.org/WAI/WCAG22/Techniques/) published by W3C.
- [x] update links in sc-urls.json, include WCAG 2.2 AAA as well as A & AA
- [x] update links in rues & ruleunits (test cases)
- [x] update links to Unit Tests that moved in Toolkit

**_example_** code for each rule help file:
`### About this requirement`
`* [WCAG technique H24](https://www.w3.org/WAI/WCAG21/Techniques/html/H24)`

would change to :
`### About this requirement`
`* [WCAG technique H24 Providing text alternatives for the area elements of image maps](https://www.w3.org/WAI/WCAG22/Techniques/html/H24)`

All ~170 existing rule Help files that reference techniques and other resources are in 
[... /accessibility-checker-engine/help-v4/en-US/ ...](https://github.com/IBMa/equal-access/tree/master/accessibility-checker-engine/help-v4/en-US)

**Comment by philljenkins** [comment]
@shunguoy ready for your review

**Comment by philljenkins** [comment]
One of the link changes had a missing ">", which caused a test case to fail in the build.  Fix and now passing.  Ready for QA

**Comment by philljenkins** [comment]
When checking the Rule sets page that is generated, found missing definitions for 3 new WCAG 2.2 SCs that don't have any rules yet.
![3 3 7 and 3 3 8](https://github.com/IBMa/equal-access/assets/16193609/fde5d9af-34e9-4863-8f92-dd68c34a15fa)
